src/NATS.Client/NATS.Client.csproj

```diff
@@ -15,11 +15,22 @@
              to their build scripts -->
         <IsPackable>false</IsPackable>
 
+        <NoWarn>$(NoWarn),SA1108,SA1129,SA1408,SA1127,SA1520,SA1132,SA1302,SA1116,SA1519,SA1206,SA1506
+            ,CS8625,CS8601,SA1100,SA1517,SA1024,SA1509,SA1203,SA1002,CS8618,SA1312,CS8622,SA1117,SA1504
+            ,SA1106,SA1308,SA1518,SA1107,SA1011,SA1122,SA1119,SA1128,SX1101,SA1137,CS8765,SA1303,SA1514
+            ,SA1304,SA1212,SA1005,SA1003,SA1124,SA1311,SA1413,SA1000,SA1310,CS8767,SA1217,SA1515,SA1214
+            ,SA1512,SX1309,SA1400,SA1204,SA1202,CS8619,CS8600,SA1004,SA1507,SA1508,SA1505,SA1001,SA1407
+            ,SA1136,SA1501,CS8602,SA1012,SA1516,SA1013,CS8604,SA1300,CS8603,SA1502,SA1028,SA1307,SA1201
+            ,SA1121,SA1027,SA1111,SA1025,SA1513,SA1401,SA1500,SA1021,SA1008,SA1009,SA1313,SA1026</NoWarn>
+
     </PropertyGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\NATS.Client.Core\NATS.Client.Core.csproj" />
-      <ProjectReference Include="..\NATS.Client.Serializers.Json\NATS.Client.Serializers.Json.csproj" />
+      <ProjectReference Include="..\NATS.Client.Simplified\NATS.Client.Simplified.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Folder Include="v1\" />
```

Comment:
Looks like this carryover from an empty 'v1' directory and can be cleaned up.

Reply:
yes spot on. Rider kindly inserted that while I wasn't looking.

Reply:
Rider is very kind. Too kind.